manifests/init.pp

```diff
@@ -200,6 +203,7 @@
   Integer[0] $api_service_worker_count = 1,
   Integer[0] $content_service_worker_timeout = 90,
   Integer[0] $api_service_worker_timeout = 90,
+  String $api_client_auth_cn = 'admin',
```

Comment:
By setting this default to `admin` means that this change can be released without breaking consumers of this, primarily Katello which uses client certificates with the CN set to `admin`.

Reply:
Should we use a stricter data type?
```suggestion
  Stdlib::Fqdn $api_client_auth_cn = 'admin',
```

AFAIK, a CN field in a cert should always be a hostname. The FQDN type also allows unqualified names (such as `admin`).